Sources/Datadog/Tracing/Propagation/HTTPHeadersWriter.swift

```diff
@@ -14,6 +14,8 @@ public class DDHTTPHeadersWriter: HTTPHeadersWriter {
         // https://github.com/DataDog/dd-trace-java/blob/4ba0ca0f9da748d4018310d026b1a72b607947f1/dd-trace-ot/src/main/java/datadog/opentracing/propagation/DatadogHttpCodec.java#L23
     }
 
+    public init() {}
```

Comment:
question regarding `OpenTracing` API: the user always needs to `init` a non-OT type (`let writer = DDHTTPHeadersWriter()` in this case), we cannot make this usage only-OT, correct?

Reply:
Yes, it's user who decides on which `OT.Writer` to use, as [there are multiple ones](https://github.com/opentracing/opentracing-swift/blob/e2b0551de2fd9a5cd9a3d8dff5390440f724283c/Sources/OpenTracing/Format.swift#L13-L34). As Datadog we only support `HTTPHeaders` 👍.